Sources/Plasma/Apps/plClient/CMakeLists.txt

```diff
@@ -241,7 +306,12 @@ target_link_libraries(
         CURL::libcurl
         "$<$<PLATFORM_ID:Darwin>:-framework Cocoa>"
         "$<$<PLATFORM_ID:Darwin>:-framework QuartzCore>"
```

Comment:
You can provide a list of platforms to PLATFORM_ID to cover both macOS and iOS:
`$<$<PLATFORM_ID:Darwin,iOS>:-framework QuartzCore>`